events/class_tests/participant_test.py

```diff
@@ -0,0 +1,57 @@
+from users.tests import user0  # noqa:F811, F401
+from ..models import Event, EventParticipant
+from .event_tests import new_event  # noqa: F401
+from django.core.exceptions import ValidationError
+
+import pytest
+
+
+def create_event_participant(event, user, is_creator):
+    return EventParticipant(event_id=event, user_id=user, is_creator=is_creator)
+
+
+@pytest.fixture
+def event_participant_creator(new_event, user0):  # noqa: F811
+    return create_event_participant(new_event, user0, True)
+
+
+@pytest.fixture
+def event_participant_not_creator(new_event, user0):  # noqa: F811
+    return create_event_participant(new_event, user0, False)
+
```

Comment:
Because of the wide use in @pytest.mark.django_db
I would suggest creating a class for all the tests in this file and placing this decorator above the class.
```suggestion
@pytest.mark.django_db
class ParticipantTests():
   pass 
```

Reply:
Done